Representable/representable
# new home page design

**changes**
- updated home page based upon designs
- changed outline buttons and button radius based on design - more blue and more rounded, in general
- added new logo on home page
- updated container to be wider on home page

**to test**
- load up the home page
- check for typos

**screenshot**
![image](https://user-images.githubusercontent.com/41943646/97944709-75135d00-1d29-11eb-9f73-c3d95b9f379b.png)
